### PR TITLE
claim perspective-python metapackage

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,44 +11,71 @@ source:
...
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
+      run:
+        - {{ pin_subpackage("perspective", max_pin="x.x.x") }}
+
+    test:
```
Comment (carterbox) [inline recipe/meta.yaml:75]: I also would have taken this opportunity to add `pip check` to the tests.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,44 +11,71 @@ source:
...
+        - perspective.table
+
+  - name: perspective-python
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
```
Comment (carterbox) [inline recipe/meta.yaml:71]: Not sure that it's necessary for the alias to have a direct dependency on python.

**Review by carterbox — COMMENTED** [review]
My understanding is that the purpose of this PR is to add an alias to this perspective package like what is done with cached_property/cached-property and importlib_resources/importlib-resources.

File: recipe/meta.yaml (hunk 2 above)
Comment (timkpaine) [reply recipe/meta.yaml:71]: strangely, this was necessary to use the `{{ PYTHON }}` jinja template in the other parts of the feedstock

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,44 +11,71 @@ source:
...
+      run:
+        - {{ pin_subpackage("perspective", max_pin="x.x.x") }}
```
[the remark this answers is not in the document]
Comment (timkpaine) [reply recipe/meta.yaml:73]: fixing in https://github.com/conda-forge/perspective-feedstock/pull/79

File: recipe/meta.yaml (hunk 2 above)
Comment (carterbox) [reply recipe/meta.yaml:71]: I also find that unexpected!

**Pull request merged**